client/layout/popup-search/index.js

```diff
@@ -42,9 +40,9 @@ export function PopUpSearch( props ) {
 			<div className="popup-search__container" onClick={ onChildClick }>
 				<HelpSearchCard
 					query={ props.searchQuery }
+					isSearching={ props.isSearching }
```

Comment:
`HelpSearchCard` is also already connected with `withInlineHelpSearchQuery` which provides the `isSearching` prop.

There are several (4+) instances of this `HelpSearchCard` component which all have a redundant `isSearching` prop like this.

Reply:
Only found 2 (addressed in https://github.com/Automattic/wp-calypso/pull/55753/commits/7fe38e9be88601b4bd9f1a833c49a33ee24ed4cc).